neuralprophet/configure.py

```diff
@@ -311,12 +311,12 @@ def __post_init__(self):
             self.trend_local_reg = False
 
         # If trend_local_reg = True
-        if self.trend_local_reg:
+        if self.trend_local_reg is True:
             log.error("trend_local_reg = True. Default trend_local_reg value set to 1")
             self.trend_local_reg = 1
 
         # If Trend modelling is global.
-        if self.trend_global_local == "global" and self.trend_local_reg:
+        if self.trend_global_local == "global" and self.trend_local_reg is True:
```

Comment:
Here,   your  change was actually correct before:
` if self.trend_global_local == "global" and self.trend_local_reg:`